rasa/core/training/interactive.py

```diff
@@ -1603,17 +1615,19 @@ def start_visualization(image_path: Text, port: int) -> None:
 
     # noinspection PyUnusedLocal
     @app.exception(NotFound)
-    async def ignore_404s(request, exception):
+    async def ignore_404s(request: Any, exception: Any) -> HTTPResponse:
```

Comment:
?
```suggestion
    async def ignore_404s(request: Request, exception: Exception) -> HTTPResponse:
```

Reply:
I have nothing to say in my defense 😄 amended